### PR TITLE
docs(console-mode): improve part of the doc

### PR DESCRIPTION
**Pull request by chentsulin**
Improved grammar and clarified some sentences.

### CODE REVIEW

File: docs/the-basics-console-mode.md [hunk 1]
```diff
@@ -5,46 +5,46 @@ title: Console Mode
 
 ## Introduction
 
-Console Mode is a productivity feature shipped with Bottender. It speeds up bot development iteration by running bots in the console. Comparing with testing your bots on existing messaging platform (e.g. Messenger, LINE), it is recommended to build, test, and debug your bot in the console mode.
+The console mode is a productivity feature shipped with Bottender. It speeds up bot development iteration by running bots in the console. Comparing with testing your bots fully on the existing messaging platforms (e.g., Messenger, LINE), we recommend building, testing, and debugging part of your bots in the console mode.
```
Comment (tw0517tw) [inline docs/the-basics-console-mode.md]:
```suggestion
The console mode is a productivity feature shipped with Bottender. It speeds up bot development iteration by running bots in the console. Comparing with testing your bots fully on existing messaging platforms (e.g., Messenger, LINE), we recommend building, testing, and debugging part of your bots in the console mode.
```

File: docs/the-basics-console-mode.md [hunk 2]
```diff
@@ -5,46 +5,46 @@ title: Console Mode
...
 
-## Running in Console Mode
+## Running in the Console Mode
```
Comment (tw0517tw) [inline docs/the-basics-console-mode.md]:
```suggestion
## Running in Console Mode
```

File: docs/the-basics-console-mode.md [hunk 3]
```diff
@@ -5,46 +5,46 @@ title: Console Mode
...
 
-In Bottender apps that created by Create Bottender App, you can pass `--console` option from npm scripts to underlying start/dev commands:
+In Bottender applications that created by Create Bottender App, you can provide the `--console` option from npm scripts to the underlying start/dev commands:
```
Comment (tw0517tw) [inline docs/the-basics-console-mode.md]:
```suggestion
In Bottender applications that created by Create Bottender App, you can provide `--console` option from npm scripts to the underlying start/dev commands:
```

File: docs/the-basics-console-mode.md [hunk 4]
```diff
@@ -5,46 +5,46 @@ title: Console Mode
...
 
 ```sh
 npm start -- --console
 npm run dev -- --console
 ```
 
-With this option, the bot will be started and run in the console:
+With the `--console` option, Bottender runs the bot in the console:
```
Comment (tw0517tw) [inline docs/the-basics-console-mode.md]:
```suggestion
With `--console` option, Bottender runs the bot in console mode:
```

File: docs/the-basics-console-mode.md [hunk 5]
```diff
@@ -5,46 +5,46 @@ title: Console Mode
...
 
 ```
 You > /payload MY_PAYLOAD
 ```
 
-## Receiving Response
+## Receiving the Responses
 
-Every responses you receive from the bot are following by `Bot >` prefix, for example:
+Every response you receive from the bot is following by the `Bot >` prefix, for example:
```
Comment (tw0517tw) [inline docs/the-basics-console-mode.md]:
```suggestion
Every response you receive from the bot is following `Bot >` prefix, for example:
```

File: docs/the-basics-console-mode.md [hunk 6]
```diff
@@ -5,46 +5,46 @@ title: Console Mode
...
 
 ```
 Bot > Hi!
 ```
 
-Except text messages, other response types will be displayed as method name and its arguments:
+Except for text message responses, Bottender displays other response types as the name and arguments of the invoked method:
```
Comment (tw0517tw) [inline docs/the-basics-console-mode.md]:
```suggestion
All other response types will be displayed by Bottender with the method name invoked and the arguments the method invoked with:
```

File: docs/the-basics-console-mode.md [hunk 7]
```diff
@@ -56,11 +56,11 @@ Bot > sendSticker with args:
 ]
 ```
 
-It's a helpful trick that let you test your bots just in the console.
+This helpful trick lets you test your bots with many different types of responses in the console.
 
-## Exiting Console Mode
+## Exiting the Console Mode
```
Comment (tw0517tw) [inline docs/the-basics-console-mode.md]:
```suggestion
## Exiting Console Mode
```

File: docs/the-basics-console-mode.md [hunk 8]
```diff
@@ -56,11 +56,11 @@ Bot > sendSticker with args:
...
 
-To exit the console mode, you can press `ctrl + c` twice or enter the following command:
+To exit the console mode, press `ctrl + c` twice or enter the following command:
```
Comment (tw0517tw) [inline docs/the-basics-console-mode.md]:
```suggestion
To exit console mode, press `ctrl + c` twice or enter the following command:
```

File: docs/the-basics-console-mode.md (hunk 5 above)
Comment (chentsulin) [reply docs/the-basics-console-mode.md]: `Every response you receive from the bot prefixes by the Bot > indicator, for example:`

File: docs/the-basics-console-mode.md (hunk 6 above)
Comment (chentsulin) [reply docs/the-basics-console-mode.md]: `Bottender displays response types as the name and arguments of the invoked method, except text message responses:`

File: docs/the-basics-console-mode.md (hunk 4 above)
Comment (chentsulin) [reply docs/the-basics-console-mode.md]: `With the `--console` option, Bottender runs the bot in Console Mode:`

**Pull request merged**